packages/components/psammead-media-player/README.md

```diff
@@ -24,13 +24,24 @@ This component to be used at any point on the page, specifically when a media pl
 | `placeholderSrc` | string | No   | `null` | `http://foobar.com/placeholder.png` |
 | `placeholderSrcset` | string | No   | - | `"https://bbc.com/300/cat.jpg 300w, https://bbc.com/450/cat.jpg 450w, https://bbc.com/600/cat.jpg 600w"` |
 | `portrait` | boolean | No   | `false` | `true` |
+| `mediaInfo` | object | Yes   | - | `{ title: 'A vertical video pretending to be a cat title',duration: '0:11',durationSpoken: '0,11',datetime: 'PT11S',type: 'video' }`|
 
 The `src` prop is required, as it tells the component what page it needs to embed.
 The `placeholderSrcset` prop is not required, as it allows image responsiveness and optimization depending on the size of the screen.
 The `title` prop is required for accessibility of the embedded iframe.
 The `portrait` prop is not required, and defaults to `false`. This is to support portrait video content in the future.
 The `showPlaceholder` boolean prop is also not required, and defaults to `true`.
 Assuming `showPlaceholder` is `true`, the `placeholderSrc` will be what image to display as the placeholder.
+The `mediaInfo` prop is required, and has the following properties
+
+| Argument  | Type                | Required | Default | Example         |
+|-----------|---------------------|----------|---------|-----------------|
+| `title` | string | Yes   | - | `"A vertical video pretending to be a cat title"` |
+| `datetime` | string | No| - | `"PT11S"` |
+| `duration` | string| No   | - | `"0,11"` |
+| `durationSpoken` | string | No   | - | `"0,11"` |
```

Comment:
`durationSpoken` will be a humanised version of the duration. Please can you amend the example to show this? For example: `2 minutes 11 seconds`.